### PR TITLE
Added translator comments to browseMode.py

### LINKED ISSUE #5822: browseMode.py has no translator comment on line 50
Pardon me if I am overstepping what types of issues I raise here, feel free to tell me to shut up if I shouldn't raise issues like this in the future.
In browseMode.py line 50, there is the string ui.message(_("focus mode"))
is there a reason this doesn't have a translator's comment?

### PR DESCRIPTION
**Pull request by ehollig**
### Link to issue number:
Fixes #5822

### Summary of the issue:
There were not translator comments for browse mode or focus mode.

### Description of how this pull request fixes the issue:
As discussed in #1410, all strings should have translator comments.

### Known issues with pull request:
None

### Change log entry:
None

### CODE REVIEW

**Comment by derekriemer** [comment]
This should take about a minute to review. Suggested merging strait to master unless others object. No code is changed but comments.

File: source/browseMode.py [hunk 1]
```diff
@@ -48,8 +48,10 @@ def reportPassThrough(treeInterceptor,onlyIfChanged=True):
 			nvwave.playWaveFile(sound)
 		else:
 			if treeInterceptor.passThrough:
+				# Translators: The mode to interact with controls in documents
 				ui.message(_("Focus mode"))
 			else:
+				# Translators: The mode that presents text in a flat representation that can be navigated with the cursor keys like in a text document
```
Comment (leonardder) [inline source/browseMode.py]: Please split this into two lines

File: source/browseMode.py [hunk 2]
```diff
@@ -1583,6 +1585,8 @@ def script_movePastEndOfContainer(self,gesture):
 		info.expand(textInfos.UNIT_CHARACTER)
 		container=self.getEnclosingContainerRange(info)
 		if not container:
+			# Translators: Reported when the user attempts to move to the start or end of a container (list, table, etc.) 
+			# But there is no container.
```
Comment (leonardder) [inline source/browseMode.py]: Great you split this into two lines, but i'd make the split before the left paren. You can also safely make the B for the start of this line a lower case letter.

**Review by LeonarddeR — CHANGES_REQUESTED** [review]
Agree with Derek about the merge to master. It kinda confuses me that, even though there are no translator comments in current master, the check pot test still passes. CC @michaeldcurran

**Comment by ehollig** [comment]
@leonardder, I believe the changes have been made. @feerrenrut this is ready for review

**Comment by michaelDCurran** [comment]
tests/checkpot.py contains a list of strings which are known not to have translator comments. Now that browse mode and focus mode do, these should be removed from that list in this PR.  The eventual goal is for there to be no blacklisted strings in checkpot.py

**Comment by ehollig** [comment]
Thank you @michaelDCurran for letting me know about tests/checkpot.py. I have updated this PR acordingly. I however, was not able to locate browse mode in checkpot.py.

File: source/browseMode.py [hunk 3]
```diff
@@ -1565,8 +1568,8 @@ def script_moveToStartOfContainer(self,gesture):
 		info.expand(textInfos.UNIT_CHARACTER)
 		container=self.getEnclosingContainerRange(info)
 		if not container:
-			# Translators: Reported when the user attempts to move to the start or end of a container (list, table, etc.) 
-			# But there is no container. 
+			# Translators: Reported when the user attempts to move to the start or end of a container
+						# (list, table, etc.) but there is no container.
```
Comment (feerrenrut) [inline source/browseMode.py]: There are extra indents here.

**Review by feerrenrut — CHANGES_REQUESTED** [review]

**Comment by feerrenrut** [comment]
@ehollig I think the reason that "Browse Mode" did not need to be added, is because it is used in several contexts already, and those usages have translator comments.

See `git grep -iF -C 2 '_("Browse Mode")'`

This alludes to a potential problem, while in this case the context for "Browse Mode" is the same. There could be cases where the context for a certain translated string is quite different.  We ideally should confirm that each translated string has its own independent translator comment. It seems that the current tests do not pick this up?

**Comment by ehollig** [comment]
@leonardder @feerrenrut, is there anything else I need to do with this PR?

**Review by LeonarddeR — APPROVED** [review]

**Review by feerrenrut — APPROVED** [review]

**Pull request merged**